tests/test_versioning.py

```diff
@@ -17,7 +17,10 @@ def _setup_module(rootdir, sphinx_test_tempdir):
     srcdir = sphinx_test_tempdir / 'test-versioning'
     if not srcdir.exists():
         shutil.copytree(rootdir / 'test-versioning', srcdir)
-    app = SphinxTestApp(srcdir=srcdir)
+    # parallelisation is not supported by this test case
+    # as the global variable 'doctrees' is not preserved
+    # when subprocesses finish
+    app = SphinxTestApp(srcdir=srcdir, parallel=0)
```

Comment:
Revert it.

Reply:
I want to keep this change as this specific test case really does not support parallel mode.
Setting this explicitly avoids issues when running all test cases in parallel mode (which I did a lot of testing this PR).

Reply:
Ah sorry for my "harsh" comment. Actually, it was because you had the 'parallel=4' before (but now it's fine).